vendor/lancier-monitoring/pipesens-codec.yaml

```diff
@@ -0,0 +1,57 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: sensor.js
+  # Examples (optional)
+  examples:

```

Comment:
@lmg-wibu examples of all devices throw errors, can you fix them? for the moment I commented them so I can run make validate command
`# examples: ...`